app.js

```diff
@@ -111,63 +48,43 @@ var storeHours = [
   '7:00 PM',
 ];
 
-var firstAndPiketotalCookies = 0;
-var firstAndPikeListElement = document.getElementById('firstAndPikeList');
+var headerEl = document.createElement('thead');
+var headerRow = document.createElement('tr');
+//add empty <th> for offset
+headerRow.appendChild(document.createElement('th'));
 
-for (var i = 0; i < resultsArrayPike.length; i++) {
-  var firstAndPikeElement = document.createElement('li');
-  firstAndPikeElement.textContent = storeHours[i] + ': ' + resultsArrayPike[i] + ' cookies';
-  firstAndPikeListElement.appendChild(firstAndPikeElement);
-  firstAndPiketotalCookies += resultsArrayPike[i];
+for (var i = 0; i < storeHours.length; i++) {
+  var thEl = document.createElement('th');
+  thEl.textContent = storeHours[i];
+  headerRow.appendChild(thEl);
 }
-var firstAndPikeTotal = document.createElement('li');
-firstAndPikeTotal.textContent = 'Total: ' + firstAndPiketotalCookies;
-firstAndPikeListElement.appendChild(firstAndPikeTotal);
+headerEl.appendChild(headerRow);
+tableEl.appendChild(headerEl);
 
-var seaTacTotalCookies = 0;
-var seaTacListElement = document.getElementById('seaTacList');
-for (var i = 0; i < resultsArraySeaTac.length; i++) {
-  var seaTacElement = document.createElement('li');
-  seaTacElement.textContent = storeHours[i] + ': ' + resultsArraySeaTac[i] + ' cookies';
-  seaTacListElement.appendChild(seaTacElement);
-  seaTacTotalCookies += resultsArraySeaTac[i];
-}
-var seaTacTotal = document.createElement('li');
-seaTacTotal.textContent = 'Total: ' + seaTacTotalCookies;
-seaTacListElement.appendChild(seaTacTotal);
+var totalCookies = 0;
 
-var seattleCenterTotalCookies = 0;
-var seattleCenterListElement = document.getElementById('seattleCenterList');
-for (var i = 0; i < resultsArraySeattleCenter.length; i++) {
-  var seattleCenterElement = document.createElement('li');
-  seattleCenterElement.textContent = storeHours[i] + ': ' + resultsArraySeattleCenter[i] + ' cookies';
-  seattleCenterListElement.appendChild(seattleCenterElement);
-  seattleCenterTotalCookies += resultsArraySeattleCenter[i];
-}
-var seattleCenterTotal = document.createElement('li');
-seattleCenterTotal.textContent = 'Total: ' + seattleCenterTotalCookies;
-seattleCenterListElement.appendChild(seattleCenterTotal);
+for (var i = 0; i < stores.length; i++) {
+  var currentStore = stores[i];
+  currentStore.randomCustomers();
 
-var capHillTotalCookies = 0;
-var capHillListElement = document.getElementById('capHillList');
-for (var i = 0; i < resultsArrayCapHill.length; i++) {
-  var capHillElement = document.createElement('li');
-  capHillElement.textContent = storeHours[i] + ': ' + resultsArrayCapHill[i] + ' cookies';
-  capHillListElement.appendChild(capHillElement);
-  capHillTotalCookies += resultsArrayCapHill[i];
-}
-var capHillTotal = document.createElement('li');
-capHillTotal.textContent = 'Total: ' + capHillTotalCookies;
-capHillListElement.appendChild(capHillTotal);
+  var rowEl = document.createElement('tr');
```

Comment:
from the assignment instructions: Each cookie stand location should have a separate render() method that creates and appends its row to the table